src/components/MessageAttachment.vue

```diff
@@ -209,17 +213,49 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+
 .attachment {
+	height: auto;
+    display: inline-flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    width: calc(33.3334% - 4px);
+    margin: 2px;
+	padding: 5px;
+    position: relative;
+    align-items: center;
+
+	&:hover {
+		background: var(--color-background-hover);
+		border-radius: 6px;
+	}
+}
+
+.mail-attachment-img--wrapper {
+	height: 44px;
+	width: 44px;
+	overflow: hidden;
+	display:flex;
+	justify-content: center;
 	position: relative;
-	display: inline-block;
-	width: calc(100% - 32px);
-	padding: 16px;
+	border-radius: 6px;
+
+	img {
+		transition: 0.3s;
+		opacity: 1;
+		width: 44px;
+		height: 44px;
+	}
+
+	.mail-attached-image {
+		width: 100px;
+	}
 }
 
-.attachment:hover,
-.attachment span:hover {
-	background-color: var(--color-background-hover);
-	cursor: pointer;
```

Comment:
this got lost. images should appear clickable if clicking triggers a preview. this will conflict with https://github.com/nextcloud/mail/pull/7066

Reply:
Restored.

_I didn't quite understand what exactly was lost. I didn't see the difference. Unless there was no style for `.attachment span:hover.`_